x-pack/test/observability_ai_assistant_api_integration/common/users/users.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { kbnTestConfig } from '@kbn/test';
+
+const password = kbnTestConfig.getUrlParts().password!;
+
+export const editorUser: User = {
+  username: 'editor',
+  password,
+  roles: ['editor'],
+};
+
+export const viewerUser: User = {
+  username: 'viewer',
+  password,
+  roles: ['viewer'],
+};
+
+export interface User {
+  username: string;
+  password: string;
+  roles: string[];
+}
```

Comment:
Nit: Could you declare this above their usages?

Reply:
Will do